lib/ioki/apis/operator_api.rb

```diff
@@ -544,7 +544,27 @@ class OperatorApi
         path:        [
           API_BASE_PATH, 'admin', 'admin_notifications', 'unread_count'
         ],
-        model_class: Ioki::Model::Operator::AdminNotification::UnreadCount
+        model_class: Ioki::Model::Operator::AdminNotifications::UnreadCount
+      ),
+      Endpoints::Create.new(
+        :admin_notification_read_confirmation,
+        base_path:            [API_BASE_PATH, 'products', :id, 'admin_notifications'],
```

Comment:
In the API it does not seem to be nested within `products`. Maybe I'm missing something, or this might be a copy&paste error?